pipeline/src/org/labkey/pipeline/status/StatusDataRegion.java

```diff
@@ -75,8 +77,9 @@ protected void renderTable(RenderContext ctx, Writer out) throws SQLException, I
 
         String controller = SpringActionController.getControllerName(_apiAction);
         String action = SpringActionController.getActionName(_apiAction);
+        PageConfig config = HttpView.currentPageConfig();
 
-        out.write("<script type=\"text/javascript\">\n" +
+        out.write("<script type=\"text/javascript\" nonce=\"" + PageFlowUtil.filter(config.getScriptNonce()) + "\">" +

```

Comment:
`getScriptNonce()` returns an `HtmlString`, doesn't it?

```suggestion
        out.write("<script type=\"text/javascript\" nonce=\"" + config.getScriptNonce() + "\">" +

```